### PR TITLE
LRQA-70350 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/LRQA-70350

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase [hunk 1]
```diff
@@ -475,7 +475,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-40911."
+	@description = "This is a use case for LRQA-40911. Add SSO openId connect provider and assert user can logout and login with openID connect."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase:478]: "Users could login and logout using OpenID Connect provider"

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase [hunk 2]
```diff
@@ -605,7 +605,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-27622."
+	@description = "This is a use case for LRQA-27622. Assert token based SSO can be enabled."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase:608]: "Users could login using Token Based SSO"

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase [hunk 3]
```diff
@@ -678,7 +678,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-27619."
+	@description = "This is a use case for LRQA-27619. View authentication instance settings."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase:681]: "Verify that LDAP and OpenSSO portlets are present at Instance Settings"

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Hi @Kvale1733 , I reviewed the namings, and they look good, but some of them are still too technical.
In these functional tests, we would like to express the goal of each test case from the business side, rather than the technical side.
I shared a few examples that I'm thinking about, feel free to modify or apply them.
I'm also waiting for @weller0246 and @ferenconodiD to review the possible test descriptions.

**Review by weller0246 — CHANGES_REQUESTED** [review]
Fixing the review comments from @gaborlovasdotcom side will cover everything from my side, keep up the good job! :)

**Comment by Kvale1733** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase [hunk 4]
```diff
@@ -9,7 +9,7 @@ definition {
 		TestCase.setUpPortalInstance();
 	}
 
-	@description = "This is a use case for LRQA-48218."
+	@description = "This is a use case for LRQA-48218. User should be redrected to the changed landing page upon login."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase:12]: typo in redirected

**Comment by Kvale1733** [comment]
Typo has been fixed

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM!

**Review by weller0246 — APPROVED** [review]

**Review by ferenconodiD — APPROVED** [review]
LGTM!

**Comment by ferenconodiD** [comment]
ci:forward

**Pull request closed without merging**